### PR TITLE
Add Static Web Server

### PR DESCRIPTION
**Pull request by joseluisq**
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [x] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

--------------

Please take some time to answer the following questions as best you can:

- **Why is it awesome?**

Regardless of being the author (I couldn't find SWS in this repo). Several users/organizations find Static Web Server (SWS) ideal to serve their static web content with ease while keeping high-performance and low-resource blueprints.
SWS users also benefit from its cross-platform nature to run SWS in bare-metal hardware, virtualized environments like Docker containers, or Webassembly.

The project is Free software written in Rust. It has received several contributions since 2019 and is actively maintained since then.
Basically, the SWS aim is to provide a web server to serve static content with a focus on high performance, security, and easy-to-use.

- **Have you used it? For how long?**

Yes, since 2019 as a freelancer on production too for hosting several personal/private websites.
But, as said early. Dozens of users are using it too to serve their websites for different purposes in other production environments.

- **Is this in a personal or professional setup?**

It is applicable for both personal and professional setups.

- **How many devices/users/services/... do you manage with it?**

SWS powers my several private/public personal projects in a Docker cluster environment. Which is my use.
However, there are several users using it in production. I list below just a few other production users.

- http://www.itis.swiss/
- https://aptoslabs.com/
- https://wasmer.io/
- https://www.singlestore.com/
- https://mytiki.com/

**Biggest pros/cons compared to other solutions?**

**Pros**

- Easy to use: In contrast with well know servers like Nignx or Apache. SWS can be easily spun up either from the CLI, environment variables or a simple configuration file since it comes with sensitive-powerful defaults but without sacrificing performance.
- Performant, efficient, and memory safe: SWS is written in Rust, a high-performance, efficient, and safe programming language which empowers SWS to scale up in different environments like bare-metal servers, cloud-native, or even resource-constrained environments.
- Feature-rich: SWS comes with several most known-used features to increase user productivity when serving web files fast.
- Flexible: The server can be configured from CLI, environment variables, or a configuration file. But it can also be selectively used as a library.

**Cons**

- Despite being stable, it's young if compared to other long-run servers like Apache or Nginx.
- It focuses on serving static web files. So this could be limiting for other use cases.
- Some native platforms could still be not supported at this point like Windows ARM64, NetBSD, or OpenBSD just to name a few.

**Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

As I said before. Despite being the author. We found SWS useful and I believe that more people in the community could benefit from it so that's why I wanted to share it here.
If you have any questions or comments please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -742,6 +742,7 @@ _See also: [awesome-selfhosted/Remote Access](https://github.com/awesome-selfhos
 - [Hiawatha](https://www.hiawatha-webserver.org/) - An advanced and secure webserver for Unix. ([Source Code](https://gitlab.com/hsleisink/hiawatha)) `GPL-2.0` `C`
 - [Lighttpd](https://www.lighttpd.net/) - Secure, fast, compliant, and very flexible web server that has been optimized for high-performance environments. ([Source Code](https://git.lighttpd.net/lighttpd/lighttpd1.4)) `BSD-3-Clause` `C`
 - [Nginx](https://nginx.org/en/) - HTTP and reverse proxy server, mail proxy server, and generic TCP/UDP proxy server. ([Source Code](https://hg.nginx.org/nginx/file/tip)) `BSD-2-Clause` `C`
+- [Static Web Server](https://static-web-server.net/) - A cross-platform, high-performance, and asynchronous web server for static file serving. ([Source Code](https://github.com/static-web-server/static-web-server)) `Apache-2.0 OR MIT` `Rust`
```
Comment (Rabenherz112) [inline README.md:745]:
```suggestion
- [Static Web Server](https://static-web-server.net/) - A cross-platform, high-performance, and asynchronous web server for static file serving. ([Source Code](https://github.com/static-web-server/static-web-server)) `Apache-2.0/MIT` `Rust/Docker`
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
As far as I can see the Software can also be deployed via Docker ([Documentation](https://static-web-server.net/features/docker/?h=docker))

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Comment by Dennisdennis90** [comment]
Hey

File: README.md (hunk 1 above)
Comment (aghodus) [reply README.md]: aghodusi

**Pull request merged**